x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import { get, getOr, isArray } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostItem, HostsEdges } from 'x-pack/plugins/secops/server/graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyValue } from '../../../empty_value';
+import { Provider } from '../../../timeline/data_providers/provider';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: HostsEdges[];
+  loading: boolean;
+  startDate: number;
+  endDate: number;
+}
+
+type HostSummaryProps = OwnProps;
+
+export const HostSummary = pure<HostSummaryProps>(({ data, startDate, endDate, loading }) => (
+  <EuiFlexItem style={{ maxWidth: 750 }}>
```

Comment:
The `style={{` is starting to creep back into our code base without a linter forbidding it.

I once removed them all (see https://github.com/elastic/kibana/pull/25268) but didn't look for a linter rule somewhere that maybe we could try to reduce/remove them. I probably should do that.

But now searching the code I see more bits of it showing up. :-(

For this one, can we not do a `styled(EuiFlexItem)` and then set the maxWidth?

Reply:
Nice catch -- thanks for the heads up! I'll update to use a `StyledEuiFlexItem` for this. 
(I would welcome a linter for these as well :)